src/lib.rs

```diff
@@ -131,6 +133,7 @@ pub fn ttfb(input: String, allow_insecure_certificates: bool) -> Result<TtfbOutc
 }
 
 /// Initializes the TCP connection to the IP address. Measures the duration.
+// #[no_panic]
```

Comment:
I think it should be sufficient to add `#[no_panic]` only to the public top-level function. Or is this not how this macro is supposed to work?